model/notify_admin.go

```diff
@@ -25,6 +26,8 @@ const (
 	UpgradeDowngradedWorkspace         = MattermostPaidFeature("mattermost.feature.upgrade_downgraded_workspace")
 )
 
+const WorkTemplates = "mattermost.plan.work_templates"
```

Comment:
Could we make this a generic notify admin for plugin installs? This is something that is not directly related to work template but just a use case like it could also be the marketplace in the future. 

```suggestion
const InstallPlugin = "mattermost.install.plugin"
```